modules/azure/etcd/etcd.tf

```diff
@@ -49,14 +49,14 @@ resource "random_id" "storage" {
 }
 
 resource "azurerm_storage_account" "etcd_storage" {
-  name                = "${random_id.storage.hex}"
+  name                = "${var.cluster_prefix}${random_id.storage.hex}etcd"
   resource_group_name = "${var.resource_group_name}"
   location            = "${var.location}"
   account_type        = "Premium_LRS"
 }
 
 resource "azurerm_storage_container" "etcd_storage_container" {
-  name                  = "${var.cluster_name}-etcd-storage-container"
+  name                  = "${var.cluster_name}-etcd"
```

Comment:
is it worth to include 'vhd' in the name so its easier to distinguish that this is a disk?